src/terminal/ansi_escape_codes.h

```diff
@@ -0,0 +1,152 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+ #ifndef _GUAC_TERMINAL_H
+#define _GUAC_TERMINAL_H
+ #include "config.h"
+ /**
+ * Maximum Arguments
+ */
+#define GUAC_TERMINAL_MAX_ARGUMENTS 16
+ /**
+ * Erase Display
+ */
+#define GUAC_TERMINAL_ERASE_DISPLAY 2
+ /**
+ * Erase Display including scrollbar
```

Comment:
The console code that this constant is intended to represent presumably doesn't deal with the scrollbar. Whether the terminal provides scrolling at all, implements it with a scrollbar, etc. is all outside the scope of what these codes normally deal with.

What code is this for?